src/FsAutoComplete/Parser.fs

```diff
@@ -110,13 +111,15 @@ module Parser =
     rootCommand.AddOption stateLocationOption
 
 
+
     rootCommand.SetHandler(
       Func<_, _, _, Task>(fun projectGraphEnabled stateDirectory adaptiveLspEnabled ->
         let workspaceLoaderFactory =
-          if projectGraphEnabled then
-            Ionide.ProjInfo.WorkspaceLoaderViaProjectGraph.Create
-          else
-            Ionide.ProjInfo.WorkspaceLoader.Create
+          fun toolsPath ->
+            if projectGraphEnabled then
+              Ionide.ProjInfo.WorkspaceLoaderViaProjectGraph.Create(toolsPath, ProjectLoader.globalProperties)
+            else
+              Ionide.ProjInfo.WorkspaceLoader.Create(toolsPath, ProjectLoader.globalProperties)
```

Comment:
You mentioned elsewhere about passing in 'other CLI properties' to proj-info - what were you thinking about?

Reply:
Wonder if it's worth having a `--msbuild-global-properties` CLI flag that you could do like `MSBuildProperty=Foobar,AnotherProperty=false` so it's not a hardcoded list.

Reply:
ohhhhh, I see. That would be a nice addition, but to me it almost speaks to a problem in proj-info instead - we need to be able to change the global properties during runtime and re-trigger evaluation based on that. Classic example is how we hard-code Debug right now.  So instead of a CLI flag I think I'd prefer a config setting in Ionide that would map to a JSON-RPC call to set specific global properties _for a project_, which would flow into proj-info as required. This is a bit bigger of a problem space, though.